README.md

```diff
@@ -8,17 +8,121 @@ Add time column to the schema
 
 ## Configuration
 
-- **option1**: description (integer, required)
-- **option2**: description (string, default: `"myvalue"`)
-- **option3**: description (string, default: `null`)
+- **to_column**: key-value pairs of options for new column
+  - **name** name of a column. (string, required)
+  - **type** type of a column. (string, required)
+  - **unix_timestamp_unit** unit of the unix timestamp if type of the column is long. "sec", "milli" (for milliseconds), "micro" (for microseconds), or "nano" (for nanoseconds). (string, required if the type is long.)
+- **from_column** key-value pairs of options for existing column
+  - **name** name of the column. Its column type must be long, timestamp or string (string, required)
+  - **unix_timestamp_unit** unit of the unix timestamp if type of the column is long. "sec", "milli" (for milliseconds), "micro" (for microseconds), or "nano" (for nanoseconds). (enum, default: `"sec"`)
+  - **timestamp_format** if the embulk type is `string`, the column value is parsed by this timestamp_format. (string, default: `"%Y-%m-%d %H:%m:%s %Z"`)
+- **from_value** key-value pairs of options for new column
+  - **mode** mode of new column creation. "fixed_time", "incremental_time", or "upload_time". (enum, default: `"fixed_time"`)
+  - **value** value of the fixed time if mode is "fixed_time". e.g. "2016-01-01 01:01:01 UTC". (string, required if "fixed_time" mode)
+  - **from** value of the begin of the incremental time if mode is "incremental_time". e.g. "2016-01-01 00:00:00 UTC". (string, required if "incremental_time" mode)
+  - **to** value of the end of the incremental time if mode is "incremental_time". e.g. "2016-01-02 00:00:00 UTC". (string, required if "incremental_time" mode)
+  - **timestamp_format** if the embulk type is `string`, the column value is parsed by this timestamp_format. (string, default: `"%Y-%m-%d %H:%m:%s %Z"`)
 
 ## Example
 
+embulk-filter-add_time plugin allows users to add new time based column to the existing schema.
+
+### to_column option
+
+`to_column` option allows add new column has the name specified as `name` and the type specified as `type`. For example, it adds new 'time' named column specified as `to_column` to the existing schema and inserts `Timestamp` value into each the column. If the column of same name specified as `to_column`.`name` already exists in the schema, the existing column name will be changed.
```

Comment:
Yes. I changed `Timestamp` to `timestamp`. I also added some explanation for renaming policy. e1440516fcbcac936f46535cb95632e980d295eb